systemd/ua-auto-attach.service

```
@@ -1,10 +1,10 @@
 [Unit]
 Description=Ubuntu Advantage auto attach
-Before=cloud-config.service
+After=cloud-config.service
```

I think we want
```suggestion
After=cloud-config.target
Before=cloud-config.service
```

See https://github.com/canonical/cloud-init/blob/main/systemd/cloud-config.target

---

Indeed, good catch